extract.py

```diff
@@ -15,8 +15,13 @@ def T0(shared_df, annotations_df):
     '''
     Extract the required variables for task 0.
     '''
```

Comment:
Using ''' for comment instead of #. So we can expand comment to multiple lines. Smart Choice!